CHANGELOG.md

```diff
@@ -1,19 +1,25 @@
-## 2.1.0 (February 5, 2019)
+## 2.1.0 (March 25, 2019)
 
-ARCHITECTURAL:
+NOTES:
 
 * Project switched to using Go modules, while `vendor` is left for backwards build compatibility only. It is worth having a
 look at [README.md](README.md) to understand how Go modules impact build and development.
 * Project dependency of github.com/hashicorp/terraform updated from v0.10.0 to v0.11.13.
 
 FEATURES:
 
-* New Independent disk resource - `vcd_independent_disk`
-* Ability to attach independent disk in `vcd_vapp_vm` 
+* **New Resource:** disk resource - `vcd_independent_disk`
+* resource/vcd_vapp_vm ahas ability to attach independent disk in 
+* **New Resource:** vApp network - `vcd_vapp_network`
+* resource/vcd_vapp_vm has ability to use vApp network 
 
 IMPROVEMENTS:
 
-* Insert Media now supports force ejecting on running VM
+* resource/vcd_inserted_media now supports force ejecting on running VM
+
+BUG FIXES:
+* [68] Issue - added vAPP status handling when env is very fast
```

Comment:
Maybe linking the issue would make more sense.

Reply:
Like this:
```
[[68](https://github.com/terraform-providers/terraform-provider-vcd/issues/68)]
```

Reply:
Links would be nice as we already have some in 2.0.0 log below. Looking at the Terraform [changelog guide](https://www.terraform.io/docs/extend/best-practices/versioning.html#changelog-specification), they suggest adding the link at the end of sentence, e.g.:
`resource/subnet: Now better [GH-22, GH-32]`

Reply:
We have to follow Harshicorp Guidelines. So we are left with using [GH-**]